app/src/main/java/co/nimblehq/compose/crypto/ui/common/price/PriceChange.kt

```diff
@@ -25,7 +25,8 @@ import kotlin.math.abs
 fun PriceChange(
     priceChangePercentage24hInCurrency: Double,
     modifier: Modifier,
-    iconPaddingEnd: Dp
+    iconPaddingEnd: Dp,
```

Comment:
Do we still need this? 😄 Actually we already have `displayForDetailPage` so we can switch the `paddingEnd` inside this component.